server/monitor/pom.xml

```diff
@@ -105,23 +113,13 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-http</artifactId>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-io</artifactId>
-      <scope>runtime</scope>
```

Comment:
They were being marked as declared but unused.  They are pulled in by the `jetty-server` jar so I think this removal is fine.

```
[INFO] +- org.eclipse.jetty:jetty-server:jar:9.2.17.v20160517:compile
[INFO] |  +- org.eclipse.jetty:jetty-http:jar:9.2.17.v20160517:compile
[INFO] |  \- org.eclipse.jetty:jetty-io:jar:9.2.17.v20160517:compile
```